src/test/java/com/adyen/CheckoutTest.java

```diff
@@ -253,6 +257,35 @@ public void TestPaymentMethodDetails() {
                              + "}", jsonRequest);
     }
 
+    @Test
+    public void TestSepaPaymentMethodDetails() {
+        DefaultPaymentMethodDetails defaultPaymentMethodDetails=new DefaultPaymentMethodDetails();
+        defaultPaymentMethodDetails.type(TYPE_SEPA_DIRECT_DEBIT);
+        defaultPaymentMethodDetails.setSepaOwnerName("A. Schneider");
+        defaultPaymentMethodDetails.setSepaIbanNumber("DE87123456781234567890");
+
+        PaymentsRequest paymentsRequest = createPaymentsCheckoutRequest();
+        paymentsRequest.setApplicationInfo(null);
```

Comment:
This is redundant. please remove null assignment.

Reply:
We need to remove the application info because the version is always updating and the mock request is not.